README.md

```diff
@@ -28,21 +28,30 @@ pytest --cov=minutes -vvv test
 ## Example Usage
 
 ```python
-from minutes import Speaker, Minutes
+from minutes import Speaker, Minutes, Conversation
 
-minutes = Minutes(ms_per_observation=500, model='cnn')
+minutes = Minutes(parent='cnn')
 
-# Create some speakers with some audio.
-speaker1 = Speaker('speaker1')
-speaker1.add_audio('path/to/audio1.wav')
+s1.add_audio('test/fixtures/1447')  # Audio book speakers.
+s2.add_audio('test/fixtures/669')
+s3.add_audio('test/fixtures/5561')
```

Comment:
Are these 3 lines supposed to be here? I'm a bit confused since `s1` / `s2` / `s3` aren't declared yet.

Reply:
:joy: copy-pasta